Source/ACE/Entity/Container.cs

```diff
@@ -51,14 +52,54 @@ public Container(AceObject aceObject, ObjectGuid guid)
         {
         }
 
+        public virtual void ContainerPlacement(WorldObject inventoryItem)
+        {
+            var removedItem = inventory.Remove(inventoryItem.Guid);
+            var invList = inventory.OrderBy(o => o.Value.Placement);
+            uint place = 0;
+            AceObject saveableCopy;
+            foreach (var keyValuePair in invList)
```

Comment:
I knew there had to be a better way.   Thanks Mogwai!   I will also add in capacity checks as part of this work.